datafusion/core/tests/sql/expr.rs

```diff
@@ -683,6 +683,12 @@ async fn test_mathematical_expressions_with_null() -> Result<()> {
     test_expression!("asin(NULL)", "NULL");
     test_expression!("acos(NULL)", "NULL");
     test_expression!("atan(NULL)", "NULL");
+    test_expression!("sinh(NULL)", "NULL");
```

Comment:
I am sorry if I missed it in my review, but are these new functions tested with non null inputs?

Specifically it would be great to see a sql level test that invoked them with values